platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/GradlePluginDevelopmentExtension.java

```diff
@@ -147,7 +147,6 @@ public Set<SourceSet> getTestSourceSets() {
      *
      * @since 7.6
      */
-    @Incubating
```

Comment:
🤔 How is that not triggering a required change to `accepted-public-api-changes`??

Reply:
Isn't it supposed to behave that way?